index.html

```diff
@@ -3,9 +3,12 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="styles/normalize.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="styles/index.css" />
+    <link href="https://fonts.googleapis.com/css?family=Caveat|Homemade+Apple|Muli|Noto+Serif+JP|Roboto" rel="stylesheet">
   </head>
-  <body>
-    <div>
+  <body class="pagecontainer">
+    <div class="container">
 
```

Comment:
Good differentiating between the classes here.  I like how you used names that are specific and understandable!